### PR TITLE
Update material-ui + Set up graphql client support

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description
- [x] Upgrade material-ui and fix resulted bugs
- [x] Set up graphql + Apollo client
- [x] Test graphql client support
- [x] update styles use `makeStyles`

Fixes # (issue)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by karimkawambwa** [comment]
@KhadijaMahanga looks good

- good idea on the separation of queries in another file.
- I think we should remove babel-eslint and aslant from the package.json to let react-scripts handle that.

File: src/components/AnalysisContent/Actions/DownloadPDF.js [hunk 1]
```diff
@@ -111,7 +112,8 @@ const createPdfStyles = StyleSheet =>
   });
 
 const createPdf = (Document, Image, Link, Page, Text, View) => {
-  function AnalysisPDF({ classes, topic, data, takwimu }) {
+  function AnalysisPDF({ topic, data, takwimu }) {
+    const classes = useStyles();
```
Comment (kilemensi) [inline src/components/AnalysisContent/Actions/DownloadPDF.js]: This should remain `function AnalysisPDF({ classes, topic, data, takwimu })`... the `classes` actually means classes for `PDF` and not web styles.

Probably bad naming on my part... Change it to `pdfClasses` or `pdfStyles` or something better.

File: src/components/Section.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ const styles = theme => ({
   }
 });
 
-function Section({ children, classes, title, variant, ...props }) {
+function Section({ classes, children, title, variant, ...props }) {
```
Comment (kilemensi) [inline src/components/Section.js]: Why is this, `Layout.js` and `SearchDrawer.js` not using `useStyles` hook?

File: src/components/A.js [hunk 3]
```diff
@@ -37,4 +34,4 @@ A.defaultProps = {
   variant: 'inherit'
 };
 
-export default withStyles(styles)(A);
+export default A;
```
Comment (kilemensi) [inline src/components/A.js]: Lets remove this component and use the one in `hurumap-ui`.

**Pull request merged**